contracts/test/utils.ts

```diff
@@ -20,3 +20,9 @@ export async function timeTravel(provider: any, seconds: number) {
   await provider.send('evm_increaseTime', [seconds]);
   await provider.send('evm_mine', []);
 }
+
+export async function setNextBlockTimestamp(provider: any, timestamp: number, delay: number) {
```

Comment:
Instead of passing in a provider to these test helpers (`timeTravel` also, you can just use the globally available `ethers.provider`, where `ethers` is available from `import { ethers } from 'hardhat';`